tests/rt.sh

```diff
@@ -680,6 +646,7 @@ EOF
       export MACHINE_ID=${MACHINE_ID}
       export RT_COMPILER=${RT_COMPILER}
       export RTPWD=${RTPWD}
+      export INPUTDATA_ROOT=${INPUTDATA_ROOT}
```

Comment:
Technically, you only need to do `export INPUTDATA_ROOT`, since `INPUTDATA_ROOT` is already a variable defined.
But, I understand the need to be consistent above.